heartbeat/podman-etcd

```diff
@@ -1078,11 +1078,25 @@ compare_revision()
 	revision=$(attribute_node_revision get)
 	peer_revision=$(attribute_node_revision_peer)
 
-	if [ "$revision" = "" ] || [ "$revision" = "null" ] || [ "$peer_revision" = "" ] || [ "$peer_revision" = "null" ]; then
+	if [ "$revision" = "" ] || [ "$revision" = "null" ] && [ "$peer_revision" = "" ] || [ "$peer_revision" = "null" ]; then
 		ocf_log err "could not compare revisions: $NODENAME local revision: $revision, peer revision: $peer_revision"
 		return "$OCF_ERR_GENERIC"
 	fi
 
+	# When only the local revision exists, it can start normally. This ensures that it can start during node replacement events.
```

Comment:
I think this is too risky, as it be part of any normal startup, not only when an admin operates a node replacement.
There is an alternative already, the admin can set the property "force_new_cluster" on the node will ignore the revisions.